src/index.html

```diff
@@ -11,7 +11,140 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input
+            type="text"
+            name="surnameArea"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+            type="text"
+            name="nameArea"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            name="ageArea"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateBirthArea"
+          >
+        </label>
+        <label class="label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="termArea"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            required
+            name="emailArea"
+          >
+        </label>
+        <label class="label">
+          Password:
+          <input
+            type="password"
+            minlength="8"
+            maxlength="20"
+            required
+            name="passwordArea"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An intresting fact about you!</legend>
+        <label class="label">
+          Do you love cats?
+          <input
+            type="radio"
+            name="catCheck"
+            id="true"
```

Comment:
very bad name for id.. and do u really need IDs here?

Reply:
I will delete them , and for name`s I havent make any better , which name`s will be better?

Reply:
Why do you need IDs here at all?